Realm/Realm/Logging/Logger.cs

```diff
@@ -35,8 +35,11 @@ public abstract class Logger
     {
         private readonly Lazy<GCHandle> _gcHandle;
 
+        // TODO(lj): Use this default log level?
+        private static readonly LogLevel _defaultLogLevel = LogLevel.Info;
+        private static readonly LogCategory _defaultLogCategory = LogCategory.Realm;
+        private static LogCategory _logCategory = _defaultLogCategory;
```

Comment:
Why do we need those fields?

Reply:
* The `_defaultLogLevel` is currently unused, was keeping that as a reminder to myself to see if we needed it to conform with some prior behavior.
* `_defaultLogCategory` is here to set it at one place (currently used in 2 places)
* `_logCategory` was initially used for getting the current log category set but I see now that it can be removed 👍